index.html

```diff
@@ -0,0 +1,130 @@
+                        climate on
+                        Earth.</p>
+                </li>
+                <li>
+                    <p><strong>1945</strong> - Discovers a way to grown wheat twice each season, doubling wheat yields
+                    </p>
+                </li>
+                <li>
+                    <p>1953- crosses a short, sturdy dwarf breed of wheat with a high-yeidling
+                        American breed,
+                        creating a strain that responds well to fertilizer. It goes on to provide 95% of Mexico's wheat.
+                    </p>
+                    <p><strong>1962</strong> - Visits Delhi and brings his high-yielding strains of wheat to the Indian
+                        subcontinent in time to help mitigate mass starvation due to a rapidly expanding population</p>
+                    <!-- <p><strong>1968</strong> - Visits Delhi and brings his high-yielding strains of wheat to the Indian
+                        subcontinent in time to help mitigate mass starvation due to a rapidly expanding population</p> -->
+                </li>
+                <li>
+                    <p><strong>1970</strong> - receives the Nobel Peace Prize</p>
+                </li>
+                <li>
+                    <p><strong>1983</strong> - helps seven African countries dramatically increase their maize and
+                        sorghum yields
+                    </p>
+                </li>
+
+                <p>1984 - becomes a distinguished professor at Texas A&M University
+                </p>
+
+                <li>
+                    <p><strong>2005</strong>- states "we will have to double the world food supply by 2050." Argues that
+                        <!-- genetically modified crops are the only way we can meet the demand, as we run out of arable
+                        land. Says that
+                        GM crops are not inherently dangerous because "we've been genetically modifying plants and -->
+                        animals for a
+                        long time. Long before we called it science, people were selecting the best breeds."
+                    </p>
+                </li>
+                <li>
+                    <p><strong>2009</strong> - dies at the age of 95.</p>
+                </li>
+            </ul>
+
+            <blockquote>"Borlaug's life and achievement are testimony to the far-reaching contribution that one man's
+                towering
+                intellect, persistence and scientific vision can make to human peace and progress."</blockquote>
+            <cite>-- Indian Prime Minister Manmohan Singh</cite>
+        </div>
+        <p class="quote">If you have time, you should read more about this incredible human being on his
+            <a href="https://en.wikipedia.org/wi/Norman_Borlaug" id="tribute-link" target="_blank"
+                rel="noopener">Wikipedia
+                Entry</a>
```

Comment:
- [ ] this Wikipedia link is broken. remove it from here and place a workable link.

Reply:
Broken links in web pages impact negatively the user's experience and the website's SEO ranking. Nice job with finding this issue :+1:. However, as I have stated in a previous comment, I suggest you phrase your comments politely as suggestions instead of orders to sound more friendly as well as elaborate more on why you requested this change. You can refer to this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate) on how you can communicate better when doing reviews.